content/collective-communication-pt1.rst

```diff
@@ -21,6 +21,22 @@ Topic introduction here
 You really want to browse this page alongside the source of it, to see
 how this is implemented.  See the links at the to right of the page.
 
+.. table:: Point-to-point communication functions
```

Comment:
This might not be in the right file

Reply:
Indeed. I was making a summary for myself and experimenting with tables. I'll stash it in some other file.